tests/Doctrine/CustomDefaultRepositoryTest.php

```diff
@@ -16,7 +18,8 @@ public function testCustomDefaultRepository(): void
 
         $repository = $documentManager->getRepository(Simple::class);
 
-        self::assertInstanceOf(DefaultDocumentRepository::class, $repository);
-        self::assertTrue($repository->isCustomDefaultDocumentRepository());
+        $this->assertInstanceOf(DefaultDocumentRepository::class, $repository);
+        assert($repository instanceof DefaultDocumentRepository);
+        $this->assertTrue($repository->isCustomDefaultDocumentRepository());
```

Comment:
Why change `self::` to `$this->` when the rest is using static too?

Reply:
Hm, indeed. Is there any of these options preferred over the other? I think I mostly saw `$this` so far.

Reply:
Since they are static, I personally prefer the static writing.

Reply:
Since DoctrineModule and DoctrineORMModule use `$this`, I refactored this project to follow these standards.

Reply:
I'm fine with that as long as it's not mixed.